packages/devtools_app/lib/src/inspector/flutter/inspector_data_models.dart

```diff
@@ -92,6 +92,79 @@ List<double> computeRenderSizes({
   return renderSizes;
 }
 
+double computeSpaceBeforeChildren({
+  @required MainAxisAlignment mainAxisAlignment,
+  @required double freeSpace,
+  @required int childrenLength,
+}) {
+  if (childrenLength == 0) return 0.0;
+  switch (mainAxisAlignment) {
+    case MainAxisAlignment.start:
+      return 0.0;
+    case MainAxisAlignment.end:
+      return freeSpace;
+    case MainAxisAlignment.center:
+      return freeSpace * 0.5;
+    case MainAxisAlignment.spaceBetween:
+      return 0.0;
+    case MainAxisAlignment.spaceAround:
+      final spaceBetweenChildren = freeSpace / childrenLength;
+      return spaceBetweenChildren * 0.5;
+    case MainAxisAlignment.spaceEvenly:
+      return freeSpace / (childrenLength + 1);
+    default:
+      return 0.0;
+  }
+}
+
+double computeSpaceAfterChildren({
```

Comment:
Add a helper method that does the logic for all cases but start, and end and call it from both these helpers.
You could call the helper
`_computeSpaceBetweenChildren`
and have it assert if MainAxisAlignment is start or end.

Reply:
or maybe `_computeSpaceHelper`